core/tests/build_sources/templates/base.html

```diff
@@ -86,7 +86,6 @@
         isAdmin: JSON.parse('{{is_admin|js_string}}'),
         isModerator: JSON.parse('{{is_moderator|js_string}}'),
```

Comment:
Can we delete these values as well? (As we are also providing this data through userInfo controller?)

Reply:
Yes we can. I am doing it.